src/amo/constants.js

```diff
@@ -138,6 +138,7 @@ export const INSTALL_SOURCE_MOST_POPULAR = 'mostpopular';
 export const INSTALL_SOURCE_SEARCH = 'search';
 export const INSTALL_SOURCE_TOP_RATED = 'rating';
 export const INSTALL_SOURCE_TRENDING = 'hotness';
+export const INSTALL_SOURCE_TAG_SHELF = 'tag-shelf-';
```

Comment:
Could have been named "prefix" to be more explicit.

Reply:
I actually suggested that, but it was just a suggestion.

Reply:
I didn't read all of the comment, it appears.